Editor/EditorCore/PolyShapeEditor.cs

```diff
@@ -219,6 +219,10 @@ public void RebuildPolyShapeMesh(bool vertexCountChanged = false)
             if (polygon == null)
                 return;
 
+            // If strip ProBuilder scripts is called when selected this can occur
+            if (polygon.mesh == null)
+                return;
```

Comment:
This works, but I think the complete fix here would include also removing the `PolyShape` component when stripping scripts.